contracts/container/contract.go

```diff
@@ -42,6 +42,13 @@ type (
 		Size int
 	}
 
+	// ContainerInfo is the summary of all [NodeReport] claimed by all
+	// storage nodes.
+	ContainerInfo struct {
```

Comment:
type order change matters?

Reply:
not much to me, just placed where i would place it from scratch. moved it down